Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10
+
+WORKDIR /app
+
+COPY ./src /app/src
```

Comment:
Vamos passar para Inglês para uma questão de coerência, por favor

I don't agree with this line in particular.
The purpose of a `Dockerfile` is to estabilish a routine that allows us to generate an image as easily as possible and, preferably, as contained as possible.
It's to be expected that `COPY` calls are made, but, in this case, I think it'd be better for this to use _some other way_ to fetch the most recent code to the container.
It is more than acceptable to copy configs, credentials and overall secrets to the container, but even then, a choice must be made between having them passed in the `.env` and just straight up copied to the container.
As an example, I could have a machine build and push the docker image on a timer, but not want the actual source code to be there, just the "logic" and the eventual secrets.
With this said can you edit this file to match these specifications?
Thanks

Reply:
Good morning Davi. Sorry for the late reply, yesterday was the last 'sprint' day, so things were a little crazy here. I have removed the copy statements, since it was already using volumes they were really unnecessary, you are absolutely correct. I have not designed this image to hold the production code, it is really just for development. But given what you just said we could use it in production as well together with some initialization script, which would build the dist code and run the container with the code dir mapped to the expected volume dir, passing it any environment variables it needs.